client/src/components/body/Reading/Reading.js

```diff
@@ -116,14 +116,6 @@ const Reading = (props) => {
         "tw-w-full tw-flex tw-flex-col tw-align-top tw-justify-center tw-h-[40rem] tw-rounded-4xl"
       }
     >
-      <h2
-        className={
-          "tw-title tw-text-left tw-bg-white tw-w-[100%] tw-px-10 tw-text-[2.5rem]"
-        }
-      >
-        {" "}
-        Reading{" "}
-      </h2>
```

Comment:
Why did we remove this?

Reply:
If we change this, also remove the title "Reinforcement" on the reinforcement page. I kind of like having the title though